### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by Balinus**
Added "Advanced topics" in faq.md.
Added "Connecting to an external julia session on a remote machine".

### CODE REVIEW

File: docs/src/man/faq.md [hunk 1]
```diff
@@ -69,3 +69,16 @@ rapid development so do so at your own risk. That being said, the Julia nightly
...
 before documentation changes.
+
+## Advanced topics
+
+### Connecting to an external julia session on a remote machine
+
+Use case : local installation of Juno with a remote julia session
+
+1. Launch Atom/Juno on your local machine
+2. In Atom/Juno : Julia-client : connect to external process (Juno will then tell you which port to use locally --> we'll call it **PORT#2**)
```
Comment (pfitzseb) [inline docs/src/man/faq.md]: This should probably be a code block, not bold:
```
`Port#2`
```
Same below.

File: docs/src/man/faq.md [hunk 2]
```diff
@@ -69,3 +69,16 @@ rapid development so do so at your own risk. That being said, the Julia nightly
...
+3. Open a local terminal and connect to your remote server : "**ssh -R PORT#1:localhost:PORT#2 your.server.com**"
```
Comment (pfitzseb) [inline docs/src/man/faq.md]: Maybe note that that'll only work on unixes. For windows, `netsh` [should work](http://stackoverflow.com/questions/11525703/port-forwarding-in-windows).

File: docs/src/man/faq.md [hunk 3]
```diff
@@ -69,3 +69,16 @@ rapid development so do so at your own risk. That being said, the Julia nightly
 should work using the [Developer Install](). Note that this will require you to
 be on master for the Julia and Atom packages, so things will be changing likely
 before documentation changes.
+
+## Advanced topics
+
+### Connecting to an external julia session on a remote machine
+
+Use case : local installation of Juno with a remote julia session
```
Comment (pfitzseb) [inline docs/src/man/faq.md]: I'd remove the space before the semi-colon; looks strange to me. Below as well.

**Review by pfitzseb — CHANGES_REQUESTED** [review]
Thanks for writing this up!

File: docs/src/man/faq.md (hunk 3 above)
Comment (Balinus) [reply docs/src/man/faq.md]: There's no space in english between a colon and the previous word. I agree it looks strange though, but I think we should follow english punctuation rules.

File: docs/src/man/faq.md (hunk 2 above)
Comment (Balinus) [reply docs/src/man/faq.md]: Good idea. Although I can't comment on the windows procedure. I will link the thread you mention.

File: docs/src/man/faq.md (hunk 1 above)
Comment (Balinus) [reply docs/src/man/faq.md]: ok!

File: docs/src/man/faq.md (hunk 3 above)
Comment (pfitzseb) [reply docs/src/man/faq.md]: > There's no space in english between a colon and the previous word.

But that's what I said? And you *do* have a space there.

Comment (Balinus) [reply docs/src/man/faq.md]: oh! I misread and I was sure there were no space to begin with. Sorry, my bad! 
edit - In french, there's a space. Hence my constant confusion! :)

Comment (Balinus) [reply docs/src/man/faq.md]: It should be OK with the latest commit: https://github.com/JunoLab/JunoDocs.jl/pull/11/commits/8f21b877a8229c78ba38fdadc1f502161044db7c

**Review by pfitzseb — APPROVED** [review]

**Pull request merged**